4-WebApp-your-API/4-1-MyOrg/README-use-keyvault-certificate.md

```diff
@@ -0,0 +1,77 @@
+# Web app using client certificates with Microsoft.Identity.Web
+
+There are different approaches for [using certificates with Microsoft.Identity.Web](https://github.com/AzureAD/microsoft-identity-web/wiki/Certificates).
+This document will talk about the simplest option - use of Azure KeyVault and a certificate.
```

Comment:
>simplest [](http://example.com/codeflow?start=34&length=8)

dont use "simplest", people will ask question as why we dont cover the "hardest" :)